components/net_traits/image_cache_thread.rs

```diff
@@ -106,6 +99,12 @@ pub enum UsePlaceholder {
     Yes,
 }
 
+#[derive(Copy, Clone, PartialEq, Deserialize, Serialize)]
```

Comment:
Perhaps add a comment explaining why this would be No? (or is that explained elsewhere?).